### ISSUE TITLE
Improve Beta dashboard accessibility and section 508 compliance

### ISSUE THREAD

**Issue by mogul** [cloud-gov/cg-product#903]
Beta dashboard needs to be checked for section 508 compliance and have an accessibility audit.

Results should be incorporated into the Stratos-based dashboard via PR since we inherit it.

-- Updated Jan 2020 to refer to Beta only since cg-dashboard is being deprecated

**Comment by karareinsel** [comment]
Feedback from a cloud.gov customer about the lack of accessibility for colorblind people. This feedback is based on the old dashboard but the same green/red indicators (without icons) is present on the new dashboard so this feedback still applies.
_It looks like the "OK" status color is #0BAF00 and the "error" status color is #BE4900. The WebAIM contrast checker says these have a contrast ratio of 1.72 to 1 — https://webaim.org/resources/contrastchecker/?fcolor=BE4900&bcolor=0BAF00. These colors aren't being used as foreground/background in this case, but the overall point is that the colors aren't super high-contrast.
Would the team be open to using icons of different shapes for the different statuses, so that color is not the only distinguishing factor? In addition, choosing colors with a higher contrast ratio would make the dashboard easier to take in at a glance_.

**Comment by spgreenberg** [comment]
@hillaryj is meeting with an accessibility expert on 30 Jan 2020 to discuss Stratos. After this, I will look at implementing the recommendations.

**Comment by spgreenberg** [comment]
Waiting for accessibility guild recommendations. cc: @hillaryj

**Comment by hillaryj** [comment]
Last week, Kara and Robert Jolly and I met with the Stratos team - they're very open to any and all changes we recommend.

Meeting minutes are here :lock: [2020-02-12 Stratos Accessibility Meeting Minutes](https://docs.google.com/document/d/1-18UJ2EiLhWY0ownytH_xxZXLyGKNnPzElIg2o_ejJc/edit)

Takeaways:

- A lot of people theme their downstream, so they aren't wedded to the existing colors if we want to tweak them for accessibility/Section 508
- Robert will be making issues in the [Stratos repo](https://github.com/cloudfoundry/stratos/issues) and they'll create a tag or add permissions so the issues can be appropriately labeled
- The cloud.gov team will meet internally through the process (frequency TBD) and we'll meet back up with Stratos probably after the accessibility audit is complete

**Comment by hillaryj** [comment]
Next steps are: 
- Robert Jolly is starting the accessibility audit
- @karareinsel to schedule some recurring followup meetings internally with Robert and @spgreenberg

**Comment by karareinsel** [comment]
Synced with Robert today. Over the next 1-2 days, he's going to submit issues that can be categorized as quick wins. By the end of the week, he'll submit issues that are more complex. He'll notify me once the issues have been submitted so that we can review and prioritize. 

Moving forward, Robert and I will sync weekly to go over WIP and prioritization.

**Comment by iamjolly** [comment]
Update: I have filed some, but not all, [issues in the main stratos repo](https://github.com/cloudfoundry/stratos/issues/created_by/iamjolly), but I cannot add the tag `accessibility` to them (yet). 

I'll keep adding issues early next week and am happy to talk through them with folks anytime.

**Comment by karareinsel** [comment]
We'll start to address the issues Robert filed (at least the one marked as critical) as new team members come onboard over the next 3 - 4 weeks. Closing.
